src/Microsoft.Diagnostics.Monitoring.WebApi/Metrics/MetricsStore.cs

```diff
@@ -165,10 +165,10 @@ public async Task SnapshotMetrics(Stream outputStream, CancellationToken token)
 
         private static string GetMetricLabels(ICounterPayload metric, double? quantile)
         {
-            string metadata = metric.Metadata;
+            string allMetadata = metric.CombineTags();
 
             char separator = IsMeter(metric) ? '=' : ':';
-            var metadataValues = CounterUtilities.GetMetadata(metadata, separator);
+            var metadataValues = CounterUtilities.GetMetadata(allMetadata, separator);
```

Comment:
So for prometheus export, we will effectively combine meter, instrument, and instance tags into one set correct?